packages/main/src/GroupHeaderListItem.hbs

```diff
@@ -1,9 +1,12 @@
 <li
-	tabindex="{{_tabIndex}}"
+	tabindex="0"
```

Comment:
Not sure, but at least in ui5 when the user presses TAB from a group list item, the focus leaves the list. Hardcoded "0" means that if the list has more than 1 group list item, TAB will move the focus from the first to the second and this might contradict the spec. In addition the ItemNavigation works with these _tabIndex and this change might interfere the tab order.
The spec should be checked and if an example with mixed types of list items works with Arrows UP/DOWN and TAB| SHIFT + TAB according to the spec I am fine with the change.

Reply:
The spec says that it should always be 0. @elenastoyanovaa what do you think?

Reply:
it is just a copied value, leave it like it is

Reply:
The question is if the tabindex of the group item should be always 0, which means the tab will move from group item to group item, instead of leaving the List.


At least in openui5 it is not like that, the focus leaves the List of we tab from a group item (https://openui5nightly.hana.ondemand.com/entity/sap.m.List/sample/sap.m.sample.ListGrouping), so I wonder what is current behaviour.

Reply:
After we discussed it, we decided that it should not be hardcoded.